cluster-autoscaler/cloudprovider/aws/auto_scaling_groups.go

```diff
@@ -483,17 +484,27 @@ func (m *asgCache) buildAsgFromAWS(g *autoscaling.Group) (*asg, error) {
 	}
 
 	if g.MixedInstancesPolicy != nil {
-		getInstanceTypes := func(data []*autoscaling.LaunchTemplateOverrides) []string {
-			res := make([]string, len(data))
-			for i := 0; i < len(data); i++ {
-				res[i] = aws.StringValue(data[i].InstanceType)
+		getInstanceTypes := func(overrides []*autoscaling.LaunchTemplateOverrides) []string {
+			res := []string{}
+			for _, override := range overrides {
+				if override.InstanceType != nil {
+					res = append(res, *override.InstanceType)
+				}
 			}
 			return res
 		}
 
+		getInstanceTypeRequirements := func(overrides []*autoscaling.LaunchTemplateOverrides) *autoscaling.InstanceRequirements {
+			if len(overrides) == 1 && overrides[0].InstanceRequirements != nil {
+				return overrides[0].InstanceRequirements // austin: confirm expected and allowed requirements and overrides setups
```

Comment:
Question here around expected configurations for LaunchTemplateOverrides with instance requirements. Are instance requirements ever expected to differ between LTOs? Previously, when overriding instance types, we specified one instance type per LTO. However, with requirements, we can define whole, complete sets per LTO, all shared within one MixedInstancesPolicy.